features/bootstrap/schema/schema.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
This schema should probably not be inside the `features` folder. It makes sense to ship it in Behat releases for others to use it. The `features` folder is excluded from dist archives as it is used when contributing to Behat.

Reply:
@stof you are absolutely right. Where do you think we should store this file? 🤔